Editing notes using the github.dev editor.md

```diff
@@ -0,0 +1,47 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Editing notes using the github.dev editor
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+
+This guide will cover the basics of using the github.dev editor for the [[Obsidian Hub]]. This editor is great when you are making quick edits that don't need any Obsidian-specific functionality.
+
+## Opening the github.dev editor
+
+To open the editor you'll usually have two options: 
```

Comment:
I'm guessing people need to have a GitHub account, and be logged in to GitHub in their browser, for this to work???

Reply:
I've added a note about that.